core/src/trezor/ui/scroll.py

```diff
@@ -98,7 +98,20 @@ def dispatch(self, event: int, x: int, y: int) -> None:
         self.on_change()
 
     def create_tasks(self) -> Tuple[loop.Task, ...]:
-        return self.handle_input(), self.handle_rendering(), self.handle_paging()
+        tasks = (
+            self.handle_input(),
+            self.handle_rendering(),
+            self.handle_paging(),
+        )  # type: Tuple[loop.Task, ...]
```

Comment:
weirdly enough, this typechecks fine
```
def create_tasks(self) -> Tuple[loop.Task, ...]:
    return super().create_tasks() + (confirm_signal(),)
```
this fails:
```
def create_tasks(self) -> Tuple[loop.Task, ...]:
    return self.handle_input(), confirm_signal()
```
(expected Coroutine[Any, Any, Any], ..., got Coroutine[Any, Any, Any], Awaitable[Any])

i don't care enough to solve this correctly as the problem seems spurious from the looks of it, but if you have any ideas?